docs/getting_started.md

```diff
@@ -4,7 +4,7 @@ This guide will help you get started with the Solace AI Event Connector.
 
 ## Prerequisites
 
-- Python 3.10 or later
+- Python 3.10.x, 3.11.x or 3.12.x
```

Comment:
Isn't this just the same as later? are we not going to support `3.13`?

Reply:
Some Python packages in the project are not compatible with versions 3.13 and upper.

Reply:
The connector was not compatible with Version 3.13. However, I tested again and it seems that 3.13 supports the connector as well. Reverted this change.